### PR TITLE
#15 added option for remote book

### LINKED ISSUE #15: Remote Books
- Add option to "View this book on xyz" and link

### PR DESCRIPTION
**Pull request by ajrbyers**
Closes #15

### CODE REVIEW

File: migrations/0012_auto_20200728_1441.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='remote_url',
+            field=models.URLField(blank=True, help_text='Set this if you want to have your book link out to a remotewebsite rather than local formats.', null=True),
+        ),
+        migrations.AlterField(
+            model_name='book',
+            name='cover',
+            field=models.FileField(blank=True, null=True, storage=django.core.files.storage.FileSystemStorage(location='/vol/janeway/src/media'), upload_to=plugins.books.models.cover_images_upload_path),
```
Comment (mauromsl) [inline migrations/0012_auto_20200728_1441.py:30]: Should use `JanewayFileSystemStorage`

File: models.py [hunk 2]
```diff
@@ -54,6 +55,20 @@ class Book(models.Model):
...
+        null=True,
+        blank=True,
+        help_text='Set this if you want to have your book link out to a remote'
+                  'website rather than local formats.'
+    )
+    remote_label = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        help_text='Label for the remote link. If left blank will display as'
```
Comment (mauromsl) [inline models.py:68]: No whitespace in `"asview"`

File: models.py [hunk 3]
```diff
@@ -54,6 +55,20 @@ class Book(models.Model):
 
     purchase_url = models.URLField(null=True, blank=True)
 
+    remote_url = models.URLField(
+        null=True,
+        blank=True,
+        help_text='Set this if you want to have your book link out to a remote'
```
Comment (mauromsl) [inline models.py:61]: no space in `"remotewebsite"`

**Review by mauromsl — CHANGES_REQUESTED** [review]
Nice!
Just a coupe typos to sort out

File: migrations/0012_auto_20200728_1441.py (hunk 1 above)
Comment (ajrbyers) [reply migrations/0012_auto_20200728_1441.py:30]: Merge of #14 will fix this.

**Review by mauromsl — APPROVED** [review]
Cool, added a comment in #16 relevant to this. We will need to rebase that branch and recreate the migrations most likely

**Pull request merged**